README.md

```diff
@@ -39,154 +59,189 @@ Being a NoSQL database data isn't stored in the relational model, but as a JSON
               partialVin (Partition Key),
 
 #### Interactions/Integration with other AWS service
+
 The diagram illustrates that technical records integrate with Test Number service to request a unique number (systemNumber, trailorId, plateSerialNumber), before the information is saved into dynamo table. Technical record service also receives status updates from test result microservice to update a record from Provisional to Active [refer Test Result Submisson triggering Technical Record Status update](https://wiki.dvsacloud.uk/display/HVT/Test+Result+Submisson+triggering+Technical+Record+Status+update).
 ![Integration AWS](/docs/awsintegration.png)
 
 Above Diagram shows orchestration/event publishing involved within Technical Records service. Following are the approved patterns it uses:
 
 #### Orchestration
+
 ![Orchestration](./docs/orchestration.png)
 
 #### Choregraphy
-![Choregraphy](/docs/choregraphy.png      )
 
+![Choregraphy](/docs/choregraphy.png)
 
-In order to understand more about the responsibility of technical records service in the end to end design, please refer to [Ministry Plate Generation Design](https://wiki.dvsacloud.uk/display/HVT/Ministry+Plate+Generation+Design). 
-
-<br>
+In order to understand more about the responsibility of technical records service in the end to end design, please refer to [Ministry Plate Generation Design](https://wiki.dvsacloud.uk/display/HVT/Ministry+Plate+Generation+Design).
```

Comment:
Is this reference intentional? The service is pretty instrumental to the overall design - I think *most* tasks use it.  I wouldn't want yo single out the plates as the main place - it's used just about everywhere!

Reply:
I agree, it should probably not be part of this README, removing and resolving